NGS_analysis_icipe
==================

Course materials for the NGS analysis course run at icipe, Nairobi.

Course Programme
Bioinformatics approaches for NGS  analysis course, ICIPE, Nairobi, 24-28 November 2014.
 
 

### Monday

| When          | What                                           |     |
|---------------|------------------------------------------------|-----|
| 0845 - 0915hrs| Introductions and Welcome                             | DM, JA  |
| 0915 - 1000hrs| [Overview of different types of next generation sequencing]() | AF  |
| 1000 - 1040hrs| [Introduction to genomics experimental design](https://github.com/jelena121/NGS_analysis_icipe/blob/master/day%201/Experimental%20design%20-%20Jelena%20Aleksic.pdf?raw=true) | JA  |
|               | Break                                          |     |
| 1100 - 1300hrs| [Introduction to R]() | JA  |
|               | Lunch                                          |     |
| 1400 - 1530hrs| [R data structures]() | JA  |
|               | Break                                          |     |
| 1600 - 1730hrs| Small groups project time |   |


 
Tuesday:
0900 - 0945hrs         	Introduction to NGS analysis methods (Richard Smith-Unna)
0945 - 1030hrs                Approaches to sequence mapping and genotyping (Simon Martin)
1030 - 1100hrs               Tea/coffee break
1100 - 1300hrs         	Practical: Data visualisation in R using ggplot2 (Richard Smith-Unna)
1300 - 1400hrs               Lunch and networking
1400 - 1530hrs         	Practical: Unix basics and shell scripting (Richard Smith-Unna)
1530 - 1600hrs               Tea/coffee break
1600 - 1730hrs         	Small groups project time
 
Wednesday:
0900 - 0945hrs         	De novo sequence assembly (Benard Kulohoma, Richard Smith-Unna)
0945 - 1030hrs         	Approaches to transcriptome analysis (Jelena Aleksic)
1030 - 1100hrs               Tea/coffee break
1100 - 1300hrs         	Practical: De novo sequence assembly (Benard Kulohoma, Richard Smith-Unna)
1300 - 1400hrs               Lunch and networking
1400 - 1530hrs         	Practical: RNAseq QC and differential expression analysis (Jelena Aleksic)
1530 - 1600hrs               Tea/coffee break
1600 - 1730hrs         	Small groups project time
1900hrs                           Course dinner
 
Thursday:
0900 - 0945hrs         	Variant calling for disease diagnostics (Nicola Whiffin)
0945 - 1030hrs         	Introduction to GWAS and meta-analyses (Nicola Whiffin)
1030 - 1100hrs               Tea/coffee break
1100 - 1230hrs         	Practical: Variant filtering and annotation  (Nicola Whiffin)
1230 - 1330hrs               Lunch and networking
1330 - 1430hrs         	Practical: Quality control of GWAS data  (Nicola Whiffin)
1430 - 1500hrs               Tea/coffee break
1500 - 1600hrs         	Guest lecture: Best practices in science communication (Joy Owango, Training 
Centre in Communication)
1600 - 1730hrs         	Small groups project time
 
Friday:
0900 - 0945hrs         	Comparative and phylo-genomics (Simon Martin)
0945 - 1030hrs         	Evolutionary and population genomics (Simon Martin)
1030 - 1100hrs               Tea/coffee break
1100 - 1300hrs         	Practical: Population genomics and cross species comparisons
(Simon Martin, Anne Fischer)
1300 - 1400hrs               Lunch and networking
1400 - 1530hrs         	Practical: Population genomics cont. (Simon Martin, Anne Fischer)
1530 - 1600hrs               Tea/coffee break
1600 - 1640hrs         	Student presentations about research projects. 10 mins per group for
presentation and questions.
1640 - 1730hrs                Informal course discussions and certificate presentation and vote of thanks!
 
 

